web/src/components/common/UI/Table.tsx

```diff
@@ -285,7 +285,31 @@ const Table: React.FC<ITableProps> = (props) => {
     }),
     [],
   );
-
+  const ourGlobalFilterFunction = useCallback((rows: any, ids: any, query: string) => {
+    const caseInsenstiveSearchKeyWord = query.toLocaleLowerCase();
+    return rows.filter((row: any) => {
+      return (
+        row.values.busOrgId?.toLowerCase().includes(caseInsenstiveSearchKeyWord) ||
+        row.values.name?.toLowerCase().includes(caseInsenstiveSearchKeyWord) ||
+        row.values.description?.toLowerCase().includes(caseInsenstiveSearchKeyWord) || // ProjectDetail Table doesn't have description field
+        row.values.clusters?.find((a: string) =>
```

Comment:
Can you make this a value string
```suggestion
        row.values.clusters?.find((cluster: string) =>
```